src/Model/Command/UserRecommendation.php

```diff
@@ -41,46 +41,22 @@ class UserRecommendation extends AbstractCommand implements UserAwareInterface
     /** @var Boost[] */
     private $boosts = [];
 
-    private function __construct(
-        string $userId,
-        int $count,
-        string $scenario,
-        float $rotationRate,
-        int $rotationTime,
-        array $responseProperties
-    ) {
+    private function __construct(string $userId, string $scenario)
+    {
         $this->minimalRelevance = MinimalRelevance::LOW();
```

Comment:
This value of minimalRelevance will be always sent as part of the request?

Reply:
Thanks for pointing that out. I totally missed all the other attributes that are optional